src/test/java/com/razorpay/PaymentClientTest.java

```diff
@@ -934,4 +934,71 @@ public void fetchAllRefund() throws RazorpayException {
             assertTrue(false);
         }
     }
+
+    @Test
+    public void expandedDetails() throws RazorpayException {
+        JSONObject request = new JSONObject();
+        request.put("expand[]","payments");
```

Comment:
I checked the docs, the values of expand[] can be the following: "emi", "offers", "upi" and "card". With "payments" is it working fine?  Because it is a mock request, can we put any value for expand[]?

Reply:
Yes, we can put any value for expand[]

Reply:
ok, +1